projects/kit/components/pager/pager.component.ts

```diff
@@ -87,12 +93,18 @@ export class TuiPager implements OnChanges, AfterViewInit {
         this.move();
     }
 
+    private get gap(): number {
+        return (
+            parseInt(this.win.getComputedStyle(this.el).getPropertyValue('--t-gap')) || 9
```

Comment:
This is a heavy procedure, we need to find out a way to avoid it

Reply:
fixed